micropip/_commands/freeze.py

```diff
@@ -42,23 +42,23 @@ def freeze_data() -> dict[str, Any]:
 def load_pip_packages() -> Iterator[tuple[str, dict[str, Any]]]:
     return map(
         package_item,
-        filter(None, map(load_pip_package, importlib.metadata.distributions())),
+        filter(is_valid, map(load_pip_package, importlib.metadata.distributions())),
```

Comment:
Is `filter(None` just a no-op? It's there to make the diff look better?

Reply:
`filter(None, ...)` is the same as `filter(bool, ...)`. It was there to suppress the distributions that returned `None`. I included that to maintain the prior expectation that if the `url` is `None`, no package entry is generated (the `continue` in the prior implementation).

I've been using that form since Python 1.5, before `bool` was introduced in Python 2.3. It occurs to me I should start using `filter(bool, ...)` for better readability.